modules/Hoot/ui/modals/exportData.js

```diff
@@ -85,7 +85,7 @@ export default class ExportData {
             const children = folder.children || folder._children || (folder.data ? folder.data._children : []) || [];
 
             // skip if no children
-            if (!children.length === 0) continue;
+            if (children.length === 0) continue;
```

Comment:
Wow, is this a bug fix?

Reply:
yea found it in another place too. webstorm catching mistakes!